torch/csrc/jit/script/init.cpp

```diff
@@ -702,6 +703,7 @@ void initJitScriptBindings(PyObject* module) {
               tracer::getTracingState()->graph->pop_scope();
             }
             return result;
+            END_HANDLE_TH_ERRORS_PYBIND
```

Comment:
Why not the other bindings too?

Reply:
Because this is the only place where arbitrary pytorch code can actually run (like functions from aten). Other entry points, to the best of my knowledge only work with jit code.

Reply:
Because this is the only place where arbitrary pytorch code can actually run (like functions from aten). Other entry points, to the best of my knowledge only work with jit code.

Reply:
But if those functions ever call `TORCH_WARN` you still need to handle unbuffering the warnings in that case, right?

Reply:
If they call `TORCH_WARN`, then a cpp warning will be printed (in cerr).